### PR TITLE
Propagate the error in prow-cluster-operation actions instead of log.Fatalf

### PR DESCRIPTION
**Pull request by yt3liu**
*Goal*:To propagate the error when actions are called from Go instead of command line 

Test PR: https://github.com/knative/test-infra/pull/1578

### CODE REVIEW

**Comment by adrcunha** [comment]
/approve

I'll let.Chao LGTM since he's the original author.

File: testutils/clustermanager/prow-cluster-operation/main.go [hunk 1]
```diff
@@ -42,12 +44,16 @@ func main() {
...
-		actions.Create(o)
+		err = actions.Create(o)
 	case delete:
-		actions.Delete(o)
+		err = actions.Delete(o)
 	case get:
-		actions.Get(o)
+		err = actions.Get(o)
 	default:
 		log.Fatal("Must pass one of --create, --delete, --get")
```
Comment (vagababov) [inline testutils/clustermanager/prow-cluster-operation/main.go]: well why not `err = errors.New("Must pass one of --create, --delete, --get")` at this point?

Comment (yt3liu) [reply testutils/clustermanager/prow-cluster-operation/main.go]: Done. Thanks for pointing it out.

**Comment by chaodaiG** [comment]
/lgtm
Thank you for commenting on the functions

**Pull request merged**